src/ray/gcs/test/asio_test.cc

```diff
@@ -66,6 +67,13 @@ TEST_F(RedisAsioTest, TestRedisCommands) {
   redisAsyncCommand(ac, NULL, NULL, "SET key test");
   redisAsyncCommand(ac, GetCallback, nullptr, "GET key");
 
+  std::shared_ptr<RedisContext> shard_context =
+      std::make_shared<RedisContext>(io_service);
+  RAY_CHECK_OK(shard_context->Connect(std::string("127.0.0.1"),
```

Comment:
should we be using ASSERT_TRUE here instead?

cc @rkooo567

Reply:
Yeah let' use the ASSERT_TRUE

Reply:
Better?